tests/Http/Controller/PtpControllerTest.php

```diff
@@ -113,35 +109,7 @@ public function testNoImageAnnotations()
         $this->postJson($url)->assertStatus(400)->assertJson(
             fn (AssertableJson $json) =>
                 $json->where('message', 'No point annotations to convert!')
-                     ->etc()
+                    ->etc()
         );
     }
-
-    public function testBadJobGenerated()
-    {
-        //Test that if a bad job is generated a job ID is not present
-        $image = Image::factory()->create(['volume_id' => $this->volume()->id]);
-
-        $imageAnnotation = ImageAnnotation::factory()->create([
-            'image_id' => $image->id,
-            'shape_id' => Shape::pointId(),
-        ]);
-        $this->beEditor();
-        config(['ptp.ptp_fail_construct' => true]);
-        config(['ptp.ptp_storage_disk' => 'test']);
-
-        $url = '/api/v1/send-ptp-job/'.$this->volume()->id;
-
-        $this->postJson($url)->assertStatus(400)->assertJson(
-            fn (AssertableJson $json) =>
-                $json->where('message', 'Unable to create Point To Polygon conversion job!')
-                     ->etc()
-        );
-
-        //$this->volume does not update with attrs
-        $volume = Volume::where('id', $this->volume()->id)->first();
-
-        $this->assertFalse(isset($volume->attrs['ptp_job_id']));
-    }
```

Comment:
This must be reverted, I'm afraid. Last time I suggested to use `Bus::shouldReceive(...)->andThrow(\Exception::class)` instead of a test config variable.